src/w3c/headers.js

```diff
@@ -257,12 +258,22 @@ export function run(conf) {
     conf.logos = [];
   }
   if (conf.isUnofficial) {
+    if (!conf.specStatus) {
```

Comment:
This combination is not possible, due to following code abpve:
```js
  conf.isUnofficial = conf.specStatus === "unofficial";
````

Reply:
Good catch 🙇‍♂️